src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/PerformanceAnalyzerWebServer.java

```diff
@@ -15,142 +15,149 @@
 
 package com.amazon.opendistro.elasticsearch.performanceanalyzer;
 
-import com.amazon.opendistro.elasticsearch.performanceanalyzer.config.PluginSettings;
+
```

Comment:
Remove the extra line ?

Reply:
This extra line is due to spotless plugin formatting. It maintains two lines spaces between import and package.